.github/workflows/ci.yml

```diff
@@ -586,6 +587,45 @@ jobs:
         name: logs_java-otel_spring-boot-native_prod
         path: artifact.tar.gz
 
+  open-telemetry-integrations:
+    runs-on: ubuntu-latest
+    needs:
+    - lint
+    - test_the_test
+    - scenarios
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install runner
+      uses: ./.github/actions/install_runner
+    - name: Pull mitmproxy image
+      run: docker pull datadog/system-tests:proxy-v1 || true
+    - name: Download Open Telemetry auto instrumentation jar
+      run: curl -o binaries/dd-java-agent.jar https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar 
```

Comment:
We need to discuss how we self-instrument with Open Telemetry or with DD Tracer.
For the meantime, to avoid making big changes, I have named the open telemetry jar after the DD jar. What will be the best way to do this?
It is also important how we are going to manage the "Open Telemetry" component and its version. For example to make the @missing_feature tags work